coeus-impl/src/main/resources/org/kuali/coeus/propdev/impl/person/ProposalPersonnelPage.xml

```diff
@@ -54,7 +54,7 @@
           p:actionParameters="PropDev-PersonnelPage-Wizard.step:0" p:iconClass="icon-user"/>
 
     <bean id="PropDev-PersonnelPage-Wizard" parent="Kc-Wizard"
-          p:fieldBindingObjectPath="addKeyPersonHelper">
+          p:fieldBindingObjectPath="addKeyPersonHelper" p:retrieveViaAjax="true" >
```

Comment:
Is this required for the wizard to work? The wizard is used in a few different places in KC and the changes seem to be both shared and specific to the key personnel usage. Will the other usages break without similar changes?